binary-search-tree/node-js/test

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-npm run test
+ npm run test 
```

Comment:
Remove the newly introduced additional space

Reply:
finished the learning summary (see above) and removed extra spaces for clean code maintenance :) Great exercise, learned heaps!